TST: Move test data for test_comp_image_quantize_level from scipy to our server

Lately devdeps been chronically failing because of this remote data test. I think the scipy server is overwhelmed or they blocked us as spammer.

```
______________ TestCompressedImage.test_comp_image_quantize_level ______________

self = <astropy.io.fits.hdu.compressed.tests.test_compressed.TestCompressedImage object at ...>

    @pytest.mark.remote_data
    def test_comp_image_quantize_level(self):
        """
        Regression test for https://github.com/astropy/astropy/issues/5969
    
        Test that quantize_level is used.
    
        """
    
        np.random.seed(42)
    
        # Basically what scipy.datasets.ascent() does.
>       fname = download_file(
            "https://github.com/scipy/dataset-ascent/blob/main/ascent.dat?raw=true"
        )

astropy/io/fits/hdu/compressed/tests/test_compressed.py:97: 
...
E       urllib.error.HTTPError: HTTP Error 429: Too Many Requests
```

I hereby propose that we make a copy of this `ascent.dat` in https://github.com/astropy/astropy-data somewhere (at first I proposed `visualization` but now I think maybe that is not the right place) and then use that in the test instead.

Please advise, @hamogu as astropy-data maintainer and @saimn / @astrofrog as io.fits maintainer. Thanks!

--- a/astropy/io/fits/hdu/compressed/tests/test_compressed.py
+++ b/astropy/io/fits/hdu/compressed/tests/test_compressed.py
@@ -82,7 +82,6 @@ class TestCompressedImage(FitsTestCase):
             assert fd[1].header["NAXIS2"] == chdu.header["NAXIS2"]
             assert fd[1].header["BITPIX"] == chdu.header["BITPIX"]
 
-    @pytest.mark.skip(reason="FIXME: https://github.com/astropy/astropy/issues/19383")
     @pytest.mark.remote_data
     def test_comp_image_quantize_level(self):
         """
@@ -96,7 +95,7 @@ class TestCompressedImage(FitsTestCase):
 
         # Basically what scipy.datasets.ascent() does.
         fname = download_file(
-            "https://github.com/scipy/dataset-ascent/blob/main/ascent.dat?raw=true",
+            "https://raw.githubusercontent.com/scipy/dataset-ascent/main/ascent.dat",
             cache=True,
             show_progress=False,
         )

